mautic/mautic
# Frequency rules data source - contact reports

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #4924
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR adds a new option for contact reports, to view frequency rules associated to a contact.
[//]: # ( As applicable: )
#### Steps to test this PR:
1. Apply PR, create a new report
2. in the Data Source dropdown select Contacts -> Frequency Rules
3. Select all the columns associated to frequency rules [Frequency, Frequency time, Frequency channel, Preferred channel, Pause from, Pause to, Frequency rule added date] + contact email: 
you should be able to see the frequency rules associated to the user.
